src/main/java/ee/tuleva/onboarding/auth/ExpiredRefreshJwtException.java

```diff
@@ -0,0 +1,8 @@
+package ee.tuleva.onboarding.auth;
+
+public class ExpiredRefreshJwtException extends RuntimeException {
```

Comment:
`ExpiredRefreshTokenException` would read better and would be implementation-agnostic (i.e. the token could be in JWT format, or some other format as well)

Reply:
FYI https://projectlombok.org/features/experimental/StandardException